wireguard/metrics.go

```diff
@@ -149,13 +149,13 @@ func NewWireguardMetricsWithShims(hostname string, newWireguardClient func() (ne
 func (collector *Metrics) getDevices() []*wgtypes.Device {
 	wgClient, err := collector.newWireguardClient()
 	if err != nil {
-		collector.logCtx.WithError(err).Error("error initializing wireguard client devices")
+		collector.logCtx.WithError(err).Warn("error initializing wireguard client devices")
```

Comment:
nit: I'd also remove the word error from this. It leads users to think the log is an error, even if the severity is warning.